.github/workflows/check_release_version.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - name: Get specific changed files
         id: changed-files-specific
-        uses: tj-actions/changed-files@v23
+        uses: tj-actions/changed-files@v41
```

Comment:
What happens if you don't update this version number?  All the repos in PSD use this check for GitHub pipelines.

Reply:
I have updated several repos for this. It is a security update.  https://www.cvedetails.com/cve/CVE-2023-51664/#

Reply:
Sounds good @yoldas .  I guess I would suggest that you document which ones you've updated and create a task to update all the remaining ones.

Reply:
I have now checked out all repos and inspected the tj-actions/changed-files lines using a simple script. All repos that use the GitHub action have their version already updated to v41 except automanual (readonly repo at v23). 
As far as I remember, the patch came from dependabot and was applied and we also manually fixed some repos.